mapo/__init__.py

```diff
@@ -1,6 +1,25 @@
 """MAPO: Model-Aware Policy Optimization in RLlib."""
 
 
+def register_all_environments():
```

Comment:
Just a heads up: RLlib is not compatible with the `gym` registry. That means that custom environments will be unknown when a worker tries to instantiate it by calling `gym.make`. 

A simple workaround that I found when trying to use the `roboschool` environments is the following:
```python
def _make_env(_):
    import gym, envs_lib
    # register custom envs in envs_lib with gym
    return gym.make(“CustomEnv”)

ray.tune.register_env(“EnvName”, _make_env)
```
So we don’t have to change anything here.

Reply:
`register_env` takes in the name of the environment and a callable which takes a single `env_config` argument and returns an environment. Envs can be registered this way right before the `tune.run` call that runs the experiments.

Reply:
thanks! will incorporate this when implementing issue #37 ...